tests/test_config_parsing/test_parser_error_collection.py

```diff
@@ -341,14 +344,79 @@ def test_summary_without_eclbase():
     )
 
 
+@pytest.mark.usefixtures("use_tmpdir")
+def test_include_non_existing_file_is_located(tmpdir):
+    assert_that_config_leads_to_error(
+        config_file_contents=dedent(
+            """
+            JOBNAME my_name%d
+            NUM_REALIZATIONS 1
+            INCLUDE does_not_exists
+            """
+        ),
+        expected_error=ExpectedErrorInfo(
+            line=4,
+            column=9,
+            end_column=24,
+            match=r"INCLUDE file:.*does_not_exists not found",
+        ),
+    )
+
+
+@pytest.mark.usefixtures("use_tmpdir")
+def test_that_include_with_too_many_args_error_is_located(tmpdir):
+    assert_that_config_leads_to_error(
+        config_file_contents=dedent(
+            """
+            JOBNAME my_name%d
+            INCLUDE something arg1 arg2 dot dotdot argn
+            """
+        ),
+        expected_error=ExpectedErrorInfo(
+            line=3,
+            column=19,
+            end_column=44,
+            match=r"Keyword:INCLUDE must have exactly one argument",
+        ),
+    )
+
+
+@pytest.mark.usefixtures("use_tmpdir")
+def test_that_include_with_too_many_args_error_is_located_indirect(tmpdir):
```

Comment:
The name of this becomes a bit unreadable as the `that` is getting in your way. Just remove `that` in this case.